packages/p2p/src/pages/my-profile/block-user/block-user-empty/__tests__/block-user-empty.spec.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import { isDesktop, isMobile } from '@deriv/shared';
+import BlockUserEmpty from '..';
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    isDesktop: jest.fn(),
+    isMobile: jest.fn(),
+}));
+
+describe('BlockUserEmpty', () => {
+    it('should render <BlockUserEmpty/> component for desktop version', () => {
```

Comment:
```suggestion
    it('should render <BlockUserEmpty/> component in desktop view', () => {
```

Reply:
what do you think about this ? :thinking:

Reply:
yeah, this is better